File: String/Odd_Even.cpp
#include <iostream> 
using namespace std; 

bool isEven(int n) { return (n % 2 == 0); } 

int main() 
{ 
	int n = 101; 
	if (isEven(n)) 
		cout << "Even"; 
	else
		cout << "Odd"; 

	return 0; 
}

File: Basic/Odd_Even.cs
using System; 

class Number { 

	public static bool isEven(int n) 
	{ 
		return (n % 2 == 0); 
	} 

	public static void Main() 
	{ 
		int n = 101; 
		if (isEven(n) == true) 
			Console.WriteLine("Even"); 
		else
			Console.WriteLine("Odd"); 
	} 
} 

